tests/services/lab_test.py

```diff
@@ -86,17 +108,222 @@ async def test_env(
 
 
 @pytest.mark.asyncio
-async def test_vols(
+async def test_build_volumes(
     factory: Factory,
     obj_factory: TestObjectFactory,
     std_result_dir: Path,
 ) -> None:
     _, user = obj_factory.get_user()
     lab_manager = factory.create_lab_manager()
+    volumes = lab_manager.build_volumes(user.username)
 
-    vols = lab_manager.build_volumes(user.username)
-    vol_str = "\n".join([f"{x}" for x in vols])
-    check_file(vol_str, std_result_dir / "volumes.txt")
+    # Unfortunately, the serialization of Kubernetes API objects is huge (it
```

Comment:
I get it but it feels less consistent.

Reply:
This was a good prompt for me to think more about this, and I came up with a serialization format that I think will work, so now hopefully we're back to consistency.